sql/database.sql

```diff
@@ -5523,6 +5525,7 @@ CREATE TABLE `pnotes` (
   `message_status` VARCHAR(20) NOT NULL DEFAULT 'New',
   `portal_relation` VARCHAR(100) NULL,
   `is_msg_encrypted` TINYINT(2) DEFAULT '0' COMMENT 'Whether messsage encrypted 0-Not encrypted, 1-Encrypted',
+  `document_id` int(11) DEFAULT NULL,
```

Comment:
As discussed, there is already a way to map document id's to pnotes. Would be best to use that rather than creating another redundant method. It is done via the setGpRelation() function. Do a grep for it to see how it's used.